tests/auth_tests/test_views.py

```diff
@@ -386,6 +386,15 @@ def test_invalid_link_if_going_directly_to_the_final_reset_password_url(self):
         response = Client().get('/reset/%s/set-password/' % uuidb64)
         self.assertContains(response, 'The password reset link was invalid')
 
+    def test_override_domain_with_password_reset(self):
+        response = self.client.post('/password_reset_override_domain/',
+                                    {'email': 'staffmember@example.com'})
+        self.assertEqual(response.status_code, 302)
+        self.assertEqual(len(mail.outbox), 1)
+        message = mail.outbox[0].body
+        domain_match = re.search(r"https?://(.*)(/.*reset/\S*)", message)
+        self.assertEqual(domain_match.groups()[0], 'custom.example.com')
```

Comment:
IMO regexp is not necessary, we can simplify this, e.g.:
```python
self.assertIn('http://custom.example.com/reset/', mail.outbox[0].body)
```

Reply:
updated assert